### ISSUE TITLE
Retrieved Data - Still being trained? - API?

### ISSUE THREAD

**Issue by Meathelix1** [openai/chatgpt-retrieval-plugin#66]
If we use this plugin I assume the data that is being pulled is also could be trained on?

Can we use this plugin with the API and does it cost more due to how it has to look through the data first?

**Comment by ivyleavedtoadflax** [comment]
Yes, it would be good to get clarification on this as it is not covered by the plugin terms here: https://openai.com/policies/plugin-terms
